src/components/Header/Header.tsx

```diff
@@ -47,16 +49,27 @@ export const Header: FC<HeaderProps> = ({ activeItem }) => {
                 Market
               </DesktopMenuItem>
             </Link>
-            <Link to='myTokens'>
-              <DesktopMenuItem
-                $active={activeItem === 'My tokens'}
-                color='additional-dark'
-                size='m'
-                weight='regular'
+            {selectedAccount
+              ? <Link to='myTokens'>
+                <DesktopMenuItem
+                  $active={activeItem === 'My tokens'}
+                  color='additional-dark'
+                  size='m'
+                  weight='regular'
+                >
+                  My tokens
+                </DesktopMenuItem>
+              </Link>
+              : <DesktopMenuItem
+                  $disabled
```

Comment:
why not just
disabled = {!selectedAccount}
?

Reply:
Because when DesktopMenuItem is not disabled it is wrapped in Link, and when it is disabled it is not wrapped in Link